qiskit_experiments/data_processing/nodes.py

```diff
@@ -122,14 +127,21 @@ def __init__(self, validate: bool = True):
             validate: If set to False the DataAction will not validate its input.
         """
         super().__init__(validate=validate)
-        self._main_axes = None
-        self._means = None
-        self._scales = None
         self._n_circs = 0
         self._n_shots = 0
         self._n_slots = 0
         self._n_iq = 0
 
+    @classmethod
+    def _default_parameters(cls) -> Options:
```

Comment:
Can you add a docstring to document what these parameters are? Even if not rendered in the documentation I think it is important to have a docstring here to explain the intent of these parameters for people that will be maintaining the code.